### DISCUSSION TITLE
Menu items not showing when open second RibbonWindow

### DISCUSSION THREAD

**Discussion by kezara** [fluentribbon/Fluent.Ribbon#1089] [Q&A] [answered]
Hello,
I've created Ribbon menu as a UserControl with IsMinimized = true, which bindes to main window view model property. Main window view is RibbonWindow, and it works like a charm, until when on secondary display I open another RibbonWindow which does not have Ribbon menu at all. When second RibbonWindow is opened and maximized clicking on Ribbon menu header expands ribbon but doesn't show menu items. When second RibbonWindow is minimized everything's working fine, and clicking on header expands ribbon and menu items are visible and functional.
Is it possible that we can't have two RibbonWindows, where one window has ribbon menu and other does not have it, opened at the same time, or I'm doing something really wrong.

**Comment by batzen** [comment]
That sounds strange.

Could you provide a sample application?

The application I am working on has hundreds of windows, some with and some without a ribbon and I never experienced such an issue.

**Reply by kezara** [reply]
Thanks for reply. I'll try to make some sample code and let you know.

**Reply by kezara** [reply]
Hello, 
Sorry for delay, but it looks that I figured it out why it doesn't working as it should to. When I work remotely via Remote Desktop I have mentioned issue, but when I'm on local computer everything works fine. Yesterday I was in office and ribbon worked just fine, today I'm remote and again I have that issue. What causes this issue, I don't know. I'll try over the weekend to make some demo app to see if I can reproduce the issue, and let you know.

**Comment by kezara** [comment]
[DemoMVVMapp.zip](https://github.com/fluentribbon/Fluent.Ribbon/files/10252481/DemoMVVMapp.zip)
Hello again,
I made small project which reproduce the error, or at least similar error. 
I've created a few ViewModels and Views to sort of copy original project where the issue is. Clicking on button Open secondary display, on secondary display will be opened new window in WindowState = Normal, in that case menu items will not be visible, but when window on secondary display is maximized menu items are normally visible. In original project, menu items are not visible when window is maximized and are visible when window is minimized. 
It looks like issue is related to WindowState property, but I don't know, maybe I'm just doing something wrong.

**Reply by batzen** [reply]
Your sample might be missing something as the secondary display never contains any ribbon controls.
Judging from the code it's just an empty view.

**Reply by kezara** [reply]
Yes, new window should be empty, and it should not have to contain ribbon controls. Issue is on main view, when open new window and if it is normal, when you click on header of ribbon control on main window, items in menu would not be visible.

**Reply by batzen** [reply]
Ah, i thought it was meant the other way around.

The ribbon buttons/tabs in the main view stay visible when i open the secondary window on my machine.
So either i still don't understand the issue, or it's not happening on my machine.

I would suggest you either create a video/gif showing your issue, so i might understand it better, or we schedule a call and we can have a look at your issue together.

This is what happens on my machine (sadly ScreenToGif can't record multiple screens):
![ribbon_1089_1](https://user-images.githubusercontent.com/1971030/208285859-dea4c369-20b9-410e-995e-40abbb75fdf1.gif)

**Comment by kezara** [comment]
https://user-images.githubusercontent.com/18076352/208296630-6b6fe04c-ca2f-48a9-b098-2123755ae35f.mp4

Here's the video, hope it helps for understanding an issue.

**Reply by batzen** [reply]
Was only able to reproduce the issue while using different DPI on the second monitor.
I have no clue what's causing this issue, but will try to fix it in v10 as soon as possible.
Created issue #1090 to track this.

**Reply by kezara** [reply]
I thought about that at one moment, and forgot to mention that my HW config is 24” full HD display and 15” full hd laptop display where scale is at 125% while 24” is at 100%. 
Now I tested, and different DPI is definitely causing the issue.

**Comment by batzen** [comment, marked as answer]
Answered by #1090
